devops/aws/build-code.yml

```diff
@@ -1,18 +1,27 @@
 ---
 
 - name: Get latest Joystream code, build it and copy binary to local
```

Comment:
Is this description still valid?

Reply:
Yes this is a case where refactoring missed updating task names. Fixed in 0c140543ce00819d346b7dc1048c61e67219aa04